src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,369 @@
+
+# Acknowledgments
+
+# Appendix
+
+## Motivation
+The PayID Easy Checkout Protocol aims to enable a consistent user experience for payers paying for goods
+or services by standardizing the interaction between merchants/non-profits and customer/donor wallets.
+Given the ability to assign arbitrary metadata to a PayID as defined in [PayID-Discovery][], there is an opportunity
+to standardize the set of interactions between merchant and payer, specifically the process by which a merchant
+directs a payer to their digital wallet to complete a payment.
+We believe this protocol will enable an improved paying experience by reducing the number
+of steps a payer must take to complete a transaction.
+
+PayID Easy Checkout also limits the engineering effort needed to implement the protocol. 
+Clients wishing to adopt this pattern should only need to implement UI-level changes in order to make the flow function 
+as intended, which may aid in expanding overall adoption, further enhancing the protocol's user experience benefits. 
+
+### Design Goals
+
+#### Minimal effort for the Payer
+
+In order for a payer to checkout using the PayID Easy Checkout protocol, the payer only needs to provide a merchant
+with their PayID Easy Checkout enabled PayID.
+
+#### No New Server-Side Software
+
+Apart from a PayID Discovery compliant PayID Server, The PayID Easy Checkout Protocol does not require server-side 
+software to be run by either the payer or merchant for a payment. The PayID server is capable of providing details 
+of where to send the payer via the PayID Discovery Protocol. Assuming the wallet used by the payer has implemented 
+support in their UI for the PayID Easy Checkout Protocol, the payer can be redirected to their wallet 
+to complete their transaction.
+
+## Example Usage
+This section shows a non-normative example of PayID Easy Checkout between a hypothetical merchant and payer. The merchant
+accepts payments using the PayID pay$merchant.com, and the payer controls the PayID alice$wallet.com.
+
+### PayID Easy Checkout Initiation
+In this example, the payer might place some items in an online shopping cart on the merchant's web-site, then choose
+to checkout.  The merchant would then render a form asking for the payer's PayID, as well as a "Checkout with PayID"
+button.  Once the payer inputs their PayID `alice$wallet.com` and clicks the "Checkout with PayID" button, the merchant
+begins the PayID Easy Checkout flow.
+
+### PayID Easy Checkout Wallet Discovery
+The merchant UI would first assemble the PayID Easy Checkout URL as defined in [PayID Easy Checkout Discovery](#payid-easy-checkout-discovery),
+yielding the URL `https://wallet.com/.well-known/webfinger?resource=payid%3Aalice%24wallet.com`. 
+The merchant UI would then [query the assembled URL](#step-2-query-payid-easy-checkout-discovery-url).
+
+The HTTP request in this example would look like this:
+    
+    GET /.well-known/webfinger?resource=payid%3Aalice%24wallet.com
```

Comment:
So is the resource officially `payid:alice$wallet.com`, and not just `alice$wallet.com`?

Reply:
yeah, see [PayID URI](https://github.com/payid-org/rfcs/blob/master/dist/spec/payid-uri.txt)